File: Reverse The Number.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int n,r,rev=0;
        scanf("%d",&n);
        while(n!=0)
        {
            r=n%10;
            n=n/10;
            rev=rev*10+r;
        }
        printf("%d\n",rev);
    }
}

File: Reverse The Number.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int n=sc.nextInt();
            int r,rev=0;
            while(n!=0)
            {
                r=n%10;
                n=n/10;
                rev=rev*10+r;
            }
            System.out.println(rev);
        }
    }
}